confluent_kafka/__init__.py

```diff
@@ -5,6 +5,9 @@
                     Message,
                     Producer,
                     TopicPartition,
+                    AdminClientImpl,
```

Comment:
given you don't really want `AdminClientImpl` exposed, I think you probably need to provide a different definition of  `__all__`.

Reply:
I'm submoduling AdminClient into .admin.

I think AdminClientImpl still needs to implicitly exported since AdminClient is a subclass

Reply:
there's the mistake there - composition is much more appropriate here I think.

Reply:
Not sure what composition buys us in this case.
The only reason there is a Python AdminClient is because it makes it easier to wrap the single-future C AdminClientImpl API into a multiple feature Python API.

Reply:
I'm confused about `__all__` above. Instead I think i meant to suggest calling `AdminClientImpl` it `_AdminClientImpl`.